### ISSUE TITLE
BotBuilder Teams

### ISSUE THREAD

**Issue by andregouveia8** [Microsoft/BotBuilder#4973]
**Hello**, 

I have one **problem to render** adaptive cards on MS Teams.

![image](https://user-images.githubusercontent.com/28810430/44102587-f676bd1e-9fe1-11e8-8f62-b362e99a7947.png)

Teams dont support Adaptive Cards?

**Comment by JasonSowers** [comment]
The official status of adaptive cards on teams is currently "Coming Soon".

**Comment by andregouveia8** [comment]
And you know when is "Coming Soon"?
Weeks, Months, Yeares?

Thanks for the reply @JasonSowers 
😄

**Comment by andregouveia8** [comment]
Do you know if hero cards and tumbnail cards works on MS Teams?

**Comment by JasonSowers** [comment]
Yes, I know at least Herocards work for sure I have not tested thumbnail cards recently, but I so no reason why they would not work

**Comment by andregouveia8** [comment]
Thanks @JasonSowers
